### DISCUSSION TITLE
Request for vcpkg (C++) support for SBOM

### DISCUSSION THREAD

**Discussion by Tim-Conrad** [aquasecurity/trivy#5913] [Ideas]
### Description

It would be great if you could integrate vcpkg support (https://vcpkg.io/en/) into trivy for generating SBOMs.

### Target

SBOM

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
We're not familiar with vcpkg. How can we identify which dependency versions are used in projects managed by vcpkg? In the case of Ruby projects, we can parse Gemfile.lock.

**Reply by Tim-Conrad** [reply]
There is a `vcpkg.json` file which has dependency versions. Example would look like this: https://github.com/staratlasjim/colyseus-cpp/blob/main/vcpkg.json

There is also a discussion on this topic and there is apparently a SPDX file created: https://github.com/microsoft/vcpkg/discussions/36078
